content/de/ecosystem/contribution/contribution.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Contributions
+description: Wie kann man sich an der Entwicklung von DigiWF beteiligen?
+category: 'Contributions'
+categoryIcon: 'mdi-gift-open-outline'
+position: 1
+---
+
+## Dokumentation
+
+### Dokumentation von Integrationsartefakten
+
+Die Dokumentation von (offiziellen) Integrations Artefakten soll einerseits in dem jeweiligen Repository erfolgen. 
+Andererseits hier in der offiziellen Gesamtdokumentation. Um nicht doppelt Inhalte pflegen zu müssen gelten folgende 
+Regeln:
+
+1. Die Dokumentation erfolgt ausschließlich in englischer Sprache. Auch in andere Übersetzungen (wie beispielsweise 
+   Deutsch) wird die englische Dokumentation eingebunden. 
+2. Die Dokumentation liegt im jeweiligen Repo im Ordner `docs`.
+3. Die Abschnitte der Dokumentation wird in einzelne Abschnitte aufgeteilt. Diese werden dann in das `readme.md` im 
+   Repo, als auch in die zentrale Dokumentation eingebunden.
+4. Für jede Sprache muss im Ordner `[Sprache]/resources/modules` eine Seite erstellt werden. Die Markdown Dateien aus 
+   den Repos werden über die Komponente `DwfMarkdownLoader` eingebunden. Element Templates können über 
+   `DwfElementTemplateDocs` dargestellt werden.
+5. Für den integrationsartefakt wird ein Eintrag auf der Seite `[Sprache]/resources/modules/modules.md` erstellt 
+   (auch hier bitte für alle Übersetzungen). Hierzu muss einer der vorhandenen `DIV` Elemente kopiert und entsprechend 
```

Comment:
Was für ein DIV Element ist hier gemeint?

Reply:
Guter Punkt. Ich habe das DIV in die Komponente gepackt und beschrieben, wie diese verwendet werden soll.